spec/unit/application/client_spec.rb

```diff
@@ -91,7 +91,7 @@
 
     # protect the unit tests against accidental --delete-entire-chef-repo from firing
     # for real during tests.  DO NOT delete this line.
-    expect(FileUtils).not_to receive(:rm_rf)
+    allow(FileUtils).to receive(:rm_rf)
```

Comment:
took me a second, but yeah you don't need the expectation here, the allow will stub it for everything which protects the other specs still, then you can override with the expectation, and it doesn't work to override an expectation with an expectation...  (mostly mentioning this for my future self if it ever comes up again..)